src/publicRPCs.ts

```diff
@@ -11,7 +11,7 @@ export const publicRPCs = {
   [ChainId.base]: "https://base.llamarpc.com",
   [ChainId.bnb]: "https://binance.llamarpc.com",
   [ChainId.metis]: "https://andromeda.metis.io/?owner=1088",
-  [ChainId.gnosis]: "https://rpc.ankr.com/gnosis",
+  [ChainId.gnosis]: "https://rpc.gnosis.gateway.fm",
```

Comment:
```suggestion
  [ChainId.gnosis]: "https://gnosis.drpc.org",
```
I think drpc is more privacy friendly and bit faster but no strong opinion
Source: [chainlist](https://chainlist.org/chain/100)

Reply:
Yes, I was hesitating whether to use the “official” one from the Gnosis doc or one marked as privacy friendly on defillama. Let's use this one 👍